File: geohelp-backend/src/main/java/ch/geohelp/geohelpbackend/data/Event.java
package ch.geohelp.geohelpbackend.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter

public class Event {
    long eventID;
    double[] longitude;
    double[] latitude;
    String passphrase;
    long startTime;
    long endTime;
    String eventName;
}

File: geohelp-organizer-app/src/app/api/event.ts
export class Event {

  public eventID: number;
  public longitude: number[];
  public latitude: number[];
  public passphrase: string;
  public startTime: number;
  public endTime: number;
  public eventName: string;


  constructor(eventID: number, longitude: number[], latitude: number[], passphrase: string, startTime: number, endTime: number, eventName: string) {
    this.eventID = eventID;
    this.longitude = longitude;
    this.latitude = latitude;
    this.passphrase = passphrase;
    this.startTime = startTime;
    this.endTime = endTime;
    this.eventName = eventName;
  }
}
